src/main/java/picard/sam/FilterSamReads.java

```diff
@@ -122,17 +124,28 @@ public String toString() {
 
     @Argument(
             doc = "SortOrder of the OUTPUT SAM or BAM file, otherwise use the SortOrder of the INPUT file.",
-            optional = true, shortName = "SO")
+            optional = true,
+            shortName = "SO")
     public SAMFileHeader.SortOrder SORT_ORDER;
 
     @Argument(
             doc = "Create .reads files (for debugging purposes)",
             optional = true)
-    public boolean WRITE_READS_FILES = true;
+    public boolean WRITE_READS_FILES = false;
 
     @Argument(doc = "SAM or BAM file to write read excluded results to",
-            optional = false, shortName = "O")
+            shortName = "O")
     public File OUTPUT;
+
+    @Argument(doc = "The tag to select from input SAM/BAM",
+            optional = true,
+            shortName = "T")
+    public String TAG;
+
+    @Argument(doc = "List of tag values to extract from Input SAM/BAM",
+            shortName = "TV",
+            optional = true)
+    public List<String> TAG_VALUE;
```

Comment:
I am curious about:
1. Why can't the `JS` argument be used here instead of adding arguments?  For your use case, this would be easier than adding a PR
2. How are array-value tags handled?

Reply:
I will have a new PR in the near future and will answer your questions there.